enhancements/arbiter-clusters.md

```diff
@@ -0,0 +1,269 @@
+---
+title: arbiter-clusters
+authors:
+  - "@eggfoobar"
+reviewers:
+  - "@tjungblu"
+  - "@patrickdillon"
+  - "@williamcaban"
+  - "@deads2k"
+  - "@jerpeter1"
+approvers:
+  - "@tjungblu"
+  - "@patrickdillon"
+  - "@williamcaban"
+  - "@jerpeter1"
+  - "@deads2k"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2024-08-27
+last-updated: 2024-08-27
+tracking-link:
+  - https://issues.redhat.com/browse/OCPEDGE-1191
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Support 2 Node + 1 Arbiter Node HA Cluster
+
+## Summary
+
+This enhancement describes an ability to install OpenShift with a control plane
+that consists of 2 normal sized nodes, and 1 node that can be less powerful than
+the recommended node size. This 1 arbiter node will only be running critical
+components for maintaining HA to allow the arbiter node size to be as small and
+as low cost as possible with in reason.
+
+## Motivation
+
+Customers at the edge are requiring a more economical solution for HA
+deployments at the edge. They can support running 2 node clusters for redundancy
+but would like the option to deploy a lower cost node as an arbiter to supply
+the 3 nodes for ETCD quorum.
+
+### User Stories
+
+- As a solutions architect for a retail organization, I want to deploy OpenShift
+  at n number of store locations at the edge with only 2 regular sized nodes and
+  1 lower cost node to maintain HA and keep compute costs down.
+- As a solutions architect for cloud infrastructures, I want to offer low cost
+  OpenShift deployments on purpose built hardware for a 2 + 1 configuration.
+- As an OpenShift cluster admin I want non-critical applications deployed to my
+  2 + 1 arbiter node cluster to not be scheduled to run on the arbiter node.
+
+### Goals
+
+- Provide a new node arbiter role type that supports HA but is not a full master
+- Support installing OpenShift with 2 regular nodes and 1 arbiter node.
+- The arbiter node hardware requirement will be lower than regular nodes.
+
+### Non-Goals
+
+The below goals are not intended to be worked on now, but might be expansion
+ideas for future features.
+
+- Running the arbiter node offsite
+- Running the arbiter node as a VM local to the cluster
+- Having a single arbiter supporting multiple clusters
+- Moving from 2 + 1 to conventional 3 node cluster
```

Comment:
the etcd / ocp topology would not change to my understanding - it remains a 3 node cluster.

Reply:
This is true, I was wondering what ability we had to change that behavior, but it seems it might be a bigger challenge. I'll remove this angle from the doc.